target/mips/op_helper_cheri.c

```diff
@@ -2194,8 +2193,9 @@ static void store_cap_to_memory(CPUMIPSState *env, uint32_t cs,
     } else {
         cheri_tag_invalidate(env, vaddr, CHERI_CAP_SIZE, retpc);
     }
-    /* Finally, store the cursor */
-    cpu_stq_data_ra(env, vaddr + 8, cursor, retpc);
+
+    cpu_stq_data(env, vaddr, pesbt);
```

Comment:
Why have these two calls dropped the `_ra` suffix/argument, but all the other functions have kept them?

Reply:
I'll put it back.  It'd be nice if there were "these can't fault" variants.